test/integration/provider.test.ts

```diff
@@ -26,6 +27,12 @@ describe('AlchemyProvider', () => {
     alchemy = new Alchemy({
       apiKey: process.env.ALCHEMY_API_KEY
     });
+    if (
+      alchemy.config.network === Network.SOLANA_MAINNET ||
+      alchemy.config.network === Network.SOLANA_DEVNET
+    ) {
+      return;
+    }
```

Comment:
Consider breaking this out into a shared NON_EVM_NETWORKS constant that we can update in one place if we add more to the list